weblate/templates/language-project.html

```diff
@@ -20,10 +20,14 @@
     <a class="dropdown-toggle" data-toggle="dropdown" href="#">
       {% trans "Tools" %} <span class="caret"></span>
     </a>
-    <ul class="dropdown-menu">
-      <li><a href="#search" data-toggle="tab">{% trans "Search" %}</a></li>
-      <li><a href="{% url 'checks' %}?language={{ language.code }}&amp;project={{ project.slug }}">{% trans "Failing checks" %}</a></li>
-      <li><a href="{% url 'checks' %}?language={{ language.code }}&amp;project={{ project.slug }}&amp;ignored=true">{% trans "Ignored checks" %}</a></li>
+  </li>
+  <li class="dropdown">
+     <a class="dropdown-toggle" type="button" id="dropdownMenuDownload" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+       <i class="fa fa-download"></i> {% trans "Download" %}
+       <span class="caret"></span>
+     </a>
+     <ul class="dropdown-menu" aria-labelledby="dropdownMenuDownload">
+       {% include "translation-download.html" %}
```

Comment:
Why removing existing search menu?

Reply:
Looks like an accident, my insurance will hopefully cover the costs.

Reply:
Still the markup seems broken for search menu, please revert it to the original.

Reply:
resolved